grid_map_core/src/GridMapMath.cpp

```diff
@@ -658,7 +658,8 @@ bool colorVectorToValue(const Eigen::Vector3i & colorVector, uint64_t & colorVal
 void colorVectorToValue(const Eigen::Vector3i & colorVector, float & colorValue)
 {
   uint64_t color = (colorVector(0) << 16) + (colorVector(1) << 8) + colorVector(2);
-  colorValue = static_cast<float>(color);
+  float * temp = reinterpret_cast<float *>(&color);
```

Comment:
What's going on here?

Reply:
`colorValue = *reinterpret_cast<float*>(&color);` thats how it looks in [ROS1](https://github.com/ANYbotics/grid_map/blob/9cbb4eb315e3c4248849d0c1d9382928d0b4c3e6/grid_map_core/src/GridMapMath.cpp#L569). I have changed to static_cast during porting because the linter was complaining, but when testing with one of the demo nodes something didn't work, so reverted it back and managed to keep the linter happy and fixed the issue I was having.